src/Components/JoinScreen/Chatbox.tsx

```diff
@@ -0,0 +1,61 @@
+import { ChangeEvent, ReactElement, useEffect, useState } from "react";
+
+import { Button } from "@mui/material";
+import socket from "../Websocket/socket";
+import { useParams } from "react-router-dom";
+
+const Chatbox = (): ReactElement => {
+  const {roomName} = useParams()
+  const [messages, setMessages] = useState<string[]>([])
+  const [roomExists, setRoomExists] = useState(false)
+  const [currentMessage, setCurrentMessage] = useState("")
+  
+  useEffect(()=>{
+    socket.emit("get_room", roomName)
+    socket.on("send_room", (room)=>{
+      if(room){
+        if(!roomExists) setRoomExists(true)
+      }
+    })
+    socket.on("message_to_client", (message)=>{
+      console.log(message, 'new Message', messages)
+      
+      setMessages((prevMessages) => {
+        return [...prevMessages, message]
+      })
+    })
+    
+    return ()=>{
+      socket.off("message_to_client")
+      socket.off("send_room")
+    }
+  },[])
+
+  const changeCurrMessage = (ev: ChangeEvent<HTMLInputElement>)=>{
+    const inputValue = ev.target.value
+    setCurrentMessage(inputValue)
+  }
+  const sendMessage = () =>{
+    socket.emit("message_to_server", [currentMessage, roomName])
+    setCurrentMessage("")
+  }
+  
+  if(!roomExists) return <div>There is no room with the name {roomName}</div>
+  
+  return (
+    <div>
+      {messages.map((message, idx)=>{
+        return (
+          <li key={idx}>
+            {message}
+          </li>
+        )
+      })}
+      <input value={currentMessage} onChange={changeCurrMessage}></input>
+      <Button onClick={sendMessage}>Send Message</Button>
```

Comment:
We should probably come up with a rule for the onChange, onClick, etc. functions. I've been doing handleChange, handleOnClick, etc. 

I think the main thing is to keep it consistent throughout.